Mikita-Davidovich/NoteList
# Task-2

### Overview
> Learning how hooks work (useState, useEffect), working with local storage
### Changes
> Implemented text editing of notes
Implemented saving changes
The updated list is saved in the local storage of the browser
### How to test
> Visit [NoteList](https://mikita-davidovich.github.io/NoteList/) page
Press the edit button
Change the value of which is written in the input field
Then click the update button
After clicking on the update button, try to reload the page
If you started to enter new text of the note but then change your mind to change it, just press the exit button and you will see what happens
